cmd/tailscale/cli/configure-ssh.go

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package cli
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/peterbourgon/ff/v3/ffcli"
+	"tailscale.com/util/lineread"
+)
+
+const tsConfigStartMark = "## BEGIN Tailscale ##"
+const tsConfigEndMark = "## END Tailscale ##"
+
+func init() {
+	configureCmd.Subcommands = append(configureCmd.Subcommands, configureSSHconfigCmd)
+}
+
+var configureSSHconfigCmd = &ffcli.Command{
+	Name:       "sshconfig",
+	ShortHelp:  "[ALPHA] Configure $HOME/.ssh/config to check Tailscale for KnownHosts",
+	ShortUsage: "sshconfig >> $HOME/.ssh/config",
+	LongHelp: strings.TrimSpace(`
+Run this command to output a ssh_config snippet that configures SSH to check
+Tailscale for KnownHosts.
+
+You can use this snippet by running: tailscale sshconfig >> $HOME/.ssh/config
+or copy and paste it into your $HOME/.ssh/config file.
+`),
+	Exec: runConfigureSSHconfig,
+	FlagSet: (func() *flag.FlagSet {
+		fs := newFlagSet("sshconfig")
+		fs.BoolVar(&sshConfigArgs.export, "export", false, "export the config snippet to stdout or modify $HOME/.ssh/config in place")
```

Comment:
this flag name and help text isn't clear what boolean true means.

Also, this doesn't work in the macOS sandbox. See:

```
bradfitz@book1pro tailscale % git grep version.IsSandboxedMacOS
cli/cert.go:    needMacWarning := version.IsSandboxedMacOS()
cli/configure-kube.go:          if version.IsSandboxedMacOS() {
cli/configure-kube.go:  if version.IsSandboxedMacOS() {
cli/configure-kube.go:                  if version.IsSandboxedMacOS() && errors.Is(err, os.ErrPermission) {
cli/file.go:                            if version.IsSandboxedMacOS() {
cli/serve_legacy.go:            if version.IsSandboxedMacOS() {
cli/up.go:              return version.IsSandboxedMacOS()
```

Reply:
Thanks I don't have the experience with the Sandboxed Mac apps so I wasn't sure.